electron/renderer/src/components/Webviews.jsx

```diff
@@ -60,7 +60,7 @@ class Webviews extends Component {
   }
 
   _getEnvironmentUrl(account, forceLogin) {
-    const envParam = decodeURIComponent(new URL(window.location).searchParams.get('env'));
+    const envParam = decodeURIComponent(new URL(window.location.href).searchParams.get('env'));
```

Comment:
Actual fix

Reply:
Let's move part of this complicated structure out into a const